eng/common/pipelines/templates/jobs/archetype-sdk-tests-generate.yml

```diff
@@ -39,7 +39,6 @@ parameters:
 - name: PreGenerationSteps
   type: stepList
   default: []
-
```

Comment:
Thanks for moving this up a level as we chatted about. You still have a deleted line here which will cause the PR to be blocked.